proto/sideloaded.proto

```diff
@@ -0,0 +1,72 @@
+// File contains definitions for side-loaded data-structures
+//
+// Because of the more distributed nature of this data
+// compared to the previous protobuf structures used in Valhalla
+// it is important to follow the protobuf schema evolution rules
+// when modifying this file
+
+syntax = "proto3";
+option optimize_for = LITE_RUNTIME;
+package valhalla_sideloaded;
+
+message IncidentsTile {
+  // Sorted list of edge_ids describing what incidents are attached to an edge_id
+  repeated EdgeToIncident edge_to_incidents = 1;
+  // Look at `edge_to_incidents` to find how to index this array
+  repeated Incident incidents = 2;
```

Comment:
i think this would be clearer with some naming changes
```
repeated Incident incidents = 1
repeated MetaData metadata = 2;
```

Reply:
`Incident` -> `IncidentLocation`?